src/main/webapp/WEB-INF/tags/instructor/results/remindButton.tag

```diff
@@ -6,8 +6,8 @@
 <%@ attribute name="remindButton" type="teammates.ui.template.InstructorFeedbackResultsRemindButton" required="true" %>
 
 <div class="remind-no-response">
-  <a href="javascript:;" data-actionlink="${remindButton.urlLink}" class="${remindButton.className}" data-toggle="modal"
-      <c:if test="${remindButton.disabled}">disabled=""</c:if> data-target="#remindModal">
+  <a href="javascript:;" data-actionlink="${remindButton.urlLink}" class="${remindButton.className}<c:if test="${remindButton.disabled}"> disabled</c:if>" data-toggle="modal"
```

Comment:
This one is quite a unique case where the link is in the `data-actionlink` instead of `href`, but let's try the same treatment.